cpp/demo/CMakeLists.txt

```diff
@@ -1,28 +1,20 @@
 cmake_minimum_required(VERSION 3.9)
 project(dolfinx-demos)
 
-# Find DOLFIN config file (not used here, but check that the demo will
-# be able to find it
-if (NOT TARGET dolfinx)
-  find_package(DOLFINX REQUIRED)
-endif()
+# Find DOLFINX config file
+find_package(DOLFINX REQUIRED)
```

Comment:
Does this cause all the dolfinx CMake config to be performed even if it had already been computed by CMake?

Reply:
It must be performed, `demo` folder is now configured as a standalone project.